asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -277,6 +278,36 @@ def get_mcs_retrofits():
     return mcs_retrofits
 
 
+def generate_hex_counts(wales_df, pc_df):
```

Comment:
or something similar.. just changing names to make it more obvious what they represent. not required, just a suggestion :) 
```suggestion
def generate_hex_counts(wales_epc, location_info):
```